### PR TITLE
Odds and ends 

### PR DESCRIPTION
**Pull request by profxj**
A modest set of edits related to work I've done to integrate
parts of PYPIT with GMOS data reduction (initial steps were
performed in IRAF).

This includes allowing for 'rows' that were completely clobbered
or missing.

Am now stating in the Install notes that numpy 1.13 is required
although this is probably not formally true.

### CODE REVIEW

File: pypit/arspecobj.py [hunk 1]
```diff
@@ -140,7 +144,7 @@ def init_exp(slf, scidx, det, fitsdict, trc_img, ypos=0.5, **kwargs):
         # Object traces
         if trc_img[sl]['nobj'] != 0:
             # Loop on objects
-            for qq in range(trc_img[sl]['traces'].shape[1]):
+            for qq in range(trc_img[sl]['nobj']):
```
Comment (rcooke-ast) [inline pypit/arspecobj.py]: I think this needs to be reinstated to the original text (for the echelle pipeline). Was it broken?

**Comment by rcooke-ast** [comment]
Don't forget to add a note in CHANGES.rst!  :-)

File: pypit/arspecobj.py (hunk 1 above)
Comment (profxj) [reply pypit/arspecobj.py]: I have returned it to as it was.
I don't think it was broken.

**Comment by rcooke-ast** [comment]
When the tests pass, feel free to merge - thanks!

**Comment by profxj** [comment]
merging.

**Pull request merged**